feature-extensions/apigee/jazz-apigee-proxy/README.md

```diff
@@ -0,0 +1 @@
+Common Lambda integration for Apigee 
```

Comment:
yes. although this feature is a add-on that admin can select if they want to install (after the jazz installation) apigee being one example here, Ben i believe this should go into the core(jazz repo) as that was my understanding. Did this change and when was this?

Reply:
I don't disagree at all that the function code should be in `jazz/core`, but since I needed to zip and deploy it for the time being it was simpler to include it in a subdirectory as a temporary measure as I mentioned in the PR comment (also I wanted to prompt this very discussion)

I can add logic that does a shallow Git clone of the main `jazz` repo and zips this one function we care about from the `core` directory.

Should it go into `jazz/core/<fn-name>` or `jazz/core/extension/<fn-name>` so it can be made explicit that this func is an optional part of the system?

Whichever you guys prefer is what I will go with.

Reply:
What is the current plan to enable `apigee`? Is this part of the installation or a post-installation step? I would imagine we will have the first option f(via install flags) while having some process for the second option (post-installation step). 

As part of the installation, `jazz` is already being cloned, hence it makes sense to me that we keep this in `jazz`. 

Regarding the right place in `jazz` - `jazz/core/extension/` makes sense. I might even try to pull this one level up to `jazz/extension`.

Reply:
i like the idea to have this part of jazz/core/extension. We need to discuss this offline

Reply:
@suryajak Based on the discussion we had, as an initial step it will be a standalone installer, that is optionally run as a post-installation step. It can later be integrated into the main installer as a flag.